sklearn/metrics/pairwise.py

```diff
@@ -127,6 +128,15 @@ def check_pairwise_arrays(
 
         .. versionadded:: 0.22
 
+    order : {'F', 'C'} or None, default=None
```

Comment:
This is a new API added in a bug fix release. We have not done this in recent memory.

If we need this `order` kwargs in the bug fix, an option is to create a private `_check_pairwise_arrays` with the `order` kwarg and have the public `check_pairwise_arrays` call it with `order=None`.

Reply:
Reverted via 576cc05b202871b9cb398da2bb2af53c9de951bc.